google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageReadFileChannelTest.java

```diff
@@ -17,7 +17,7 @@
 
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Truth is available in this repository, please use that

Reply:
If needed then change existing assertThat(), Becuase we can't have one set of test with hamcrest and another set with Truth within a same module